Library/Formula/saldl.rb

```diff
@@ -0,0 +1,47 @@
+class Saldl < Formula
+  desc "CLI downloader optimized for speed and early preview."
+  homepage "https://saldl.github.io"
+  url "https://github.com/saldl/saldl/archive/v31.tar.gz"
+  sha256 "146a980ae2109a391c7a8ab6a1c525458db9bf18f3f0477731c9b876630078b4"
+
+  head do
+    # git describe does not work with shallow clones
+    url "https://github.com/saldl/saldl.git", :shallow => false
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "asciidoc" => :build
+  # Install DTDs locally for a faster and more reliable build result
+  depends_on "docbook-xsl" => :build
+
+  depends_on "libevent"
+
+  # Add option to use keg_only curl formula instead of the version provided by OSX
```

Comment:
Would prefer not to provide this option personally, unless wider demand crops up over time.